Description: readme changed
 netbunch (1.0-1) unstable; urgency=medium
 .
   * Initial release
Author: Vincenzo (KatolaZ) Nicosia <katolaz@freaknet.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2017-09-28

--- netbunch-1.0.orig/README
+++ netbunch-1.0/README
@@ -6,7 +6,7 @@ accompanying the book:
   "Complex Networks: Principles, Methods and Application"
   V. Latora, V. Nicosia, G. Russo, Cambridge University 
   Press (2017), ISBN: 9781107103184
-  http://www.cambridge.org/gb/academic/subjects/physics/statistical-physics/complex-networks-principles-methods-and-applications?format=HB
+  https://www.cambridge.org/9781107103184
 
 NetBunch includes all the programs implementing the algorithms
 described in the Appendices of the book, and several more. If you use
